src/templates/bootstrap/components/fieldset.html

```diff
@@ -0,0 +1,10 @@
+<div [formGroup]="form" class="form-group">
+    <fieldset>
+        <legend *ngIf="component.settings.legend">{{ component.settings.legend }}</legend>
+        <div *ngFor="let control of component.control.controls;">
```

Comment:
You should only need to use the ```<formio-components>``` here. 

Please refer to https://github.com/formio/ng2-formio/blob/master/src/templates/bootstrap/components/columns.html